chunks/chunks.go

```diff
@@ -196,6 +196,78 @@ func (w *Writer) write(b []byte) error {
 	return err
 }
 
+// MergeOverlappingChunks removes the samples whose timestamp is overlapping.
+// The first appearing sample is retained in case there is overlapping.
```

Comment:
Yes, you are right, the last sample is retained. I had made it such that the last samples are retained everywhere (compaction and query). So I guess update the comment rather than change the implementation?

Reply:
What is the desired behaviour? Keep the first samples or last samples?

Reply:
Keep one :P Don't care which.

Reply:
I updated the comment. We keep the last samples.